certificates/.gitignore

```diff
@@ -0,0 +1,2 @@
+!.gitignore
+*
```

Comment:
I'd just ignore the entire directory and create it. This is a bit strange to me, though inconsequentual.